Content/OfficialGCCodeSupport.md

```diff
@@ -56,20 +56,44 @@ As of May 2019, GC/Code has a record of ≈3600 projects and ≈3000 users (350+
 
 To bring [GC/Code](https://gccode.ssc-spc.gc.ca) as an officially supported source control solution for the Government of Canada's developer community.
 
-### Improved Feature Support
+### Benefits to ESDC Business Lines
 
-With the current (as is) configuration of GC/Code, there are a few additional features that could be enabled to increase the viability of the tool:
+#### IITB
+
+* Strengthen relationship between IT and business through a centralized collaboration platform
+* Improve the quality and security of code with automated testing (e.g. static code analysis to detect hard coded passwords)
+* Accelerate the rate of application development and deployments through a modern tool (in constant evolution)
+* Easily find and reuse source code from other projects, including from other departments
+* Receive notifications and send actions from instant messaging platform (e.g. Slack)
+* Obtain metrics/statistics for reporting and continuous improvement
+* Edit files without leaving the browser
+
+#### Existing Development Projects (e.g. Old Age Security)
+
+* Benefit from a common development platform:
+  * Leverage functionalities covering the full development lifecycle from a single tool (code, issues/tasks, tests, etc.)
+  * Facilitate adoption/training by staff
```

Comment:
suggest adding "... in particular the automation of builds, tests, and deployment"

Reply:
Great suggestion! Added as a separate commit after the PR.